Dockerfile.aarch64

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
+
+# Adding rust binaries to PATH.
+ENV PATH="$PATH:/root/.cargo/bin"
+
+RUN apt-get update
+
+RUN apt-get -y install gcc
+
+# kcov dependencies
+RUN apt-get -y install cmake g++ pkg-config jq
+RUN apt-get -y install libcurl4-openssl-dev libelf-dev libdw-dev binutils-dev libiberty-dev
+
+# Installing rustup.
+RUN apt-get -y install curl
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# Installing rust tools used by the rust-vmm CI.
+RUN rustup component add rustfmt
+# clippy is not yet supported in stable rust for aarch64
+# Should be available starting with 12 April 2019
+# RUN rustup component add clippy
+RUN cargo install cargo-kcov
+
+# Installing other rust targets
+RUN rustup target add aarch64-unknown-linux-musl
+
+# Installing kcov.
```

Comment:
It would make more sense to have `RUN cargo install cargo-kcov` after that comment, not before.

Reply:
```RUN cargo install cargo-kcov``` has a dependency on python so I can't move that command before.